libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/utils/RouteUtils.kt

```diff
@@ -219,20 +219,15 @@ class RouteUtils {
     fun findCurrentBannerInstructions(
         currentStep: LegStep?,
         stepDistanceRemaining: Double
-    ): BannerInstructions? {
-        currentStep?.bannerInstructions()?.let {
+    ) = currentStep?.bannerInstructions()?.let {
```

Comment:
return value is not obvious 
```suggestion
    ): BannerInstructions? = currentStep?.bannerInstructions()?.let {
```

Reply:
👍